src/lints/trait_unsafe_added.ron

```diff
@@ -50,4 +50,8 @@ SemverQuery(
     },
     error_message: "A publicly-visible trait became `unsafe`, so implementing it now requires an `unsafe impl` block.",
     per_result_error_template: Some("trait {{join \"::\" path}} in file {{span_filename}}:{{span_begin_line}}"),
+    witness: (
+        hint_template: r#"struct MyStruct;
+impl {{join "::" path}} for MyStruct {...};"#,
```

Comment:
Canonically, we want the type or value closest to the breakage to be called `Witness` or `witness` as appropriate. We want users to get used to seeing the name and looking for the breakage next to it.
```suggestion
        hint_template: r#"struct Witness;
impl {{join "::" path}} for Witness {...}"#,
```